README.md

```diff
@@ -1,10 +1,56 @@
 # 2022-SurfingPenguin's-Icepace
```

Comment:
you probably want to correct this too - btw it doesn't have to follow that format, you could just write Icepace or whatever else you want